scm_repo_link.go

```diff
@@ -97,6 +99,20 @@ type SCMRepoLinkReviewAppsResponse struct {
 	ReviewApps []*ReviewApp `json:"review_apps"`
 }
 
+type SCMRepoLinkPullRequestResponse struct {
+	Pull RepoLinkPullRequest `json:"pull"`
```

Comment:
question: I know it's not related to this pull request, but why does the API only returns `pull` and not `pull_request`?

Reply:
Well, as the `/pulls` endpoint returns a `pulls` attribute, and not `pull_requests`, I went for `pull`!